LICENSE-MIT

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Zengo X
+Copyright (c) 2023
```

Comment:
The terms of MIT licence explicitly forbid you to do it. Instead we can add other contributors (like you or dfns) here

Reply:
Good call, I was hesitating what I should do here

Reply:
Updated